etsin_finder/frontend/js/components/dataset/data/idaResourcesV2/iconButton.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
```

Comment:
By the name of the component this sounds like general component more than download specific button. Not sure if we have any icon buttons other than this though.

Reply:
There's quite heavy amount of props. This also suggest that component would be general and there could opportunity to make more specific component that defines some details more specific to download case. This kind of structure would produce more readable code and more reusable components.